### DISCUSSION TITLE
Ability to retrieve evaluation value from Threshold expression

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73288] [Alerting]
Through the `$values` variable, we are able to access the result of all the queries/expressions in an alert.

However, we are not able to retrieve the actual value defined in a Threshold expression. Only the result (one or zero) is visible.

It would be useful to have the actual evaluation value from a Threshold expression so we can use it in a template (e.g. to dynamically define a summary that goes to the notification).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82566. Feel free to continue the discussion around this there. Thank you!
